src/encoder.rs

```diff
@@ -1770,7 +1778,9 @@ fn encode_partition_topdown(seq: &Sequence, fi: &FrameInvariants, fs: &mut Frame
 
     match partition {
         PartitionType::PARTITION_NONE => {
-            let part_decision = if !rdo_output.part_modes.is_empty() {
+            // FIXME: For now always redo RDO since prior RDO may have relied on inaccurate
+            // neighbours
+            let part_decision = if !rdo_output.part_modes.is_empty() && false {
```

Comment:
So, I am thinking that good chance of replacing 'false' here with sth better would be checking whether rdo_output.part_modes[0].bsize (does not exist currently) ==  current bsize.